web/pages/feeds/opml.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+  import {
+    creating,
+    exportedCategories,
+    exportedFeeds,
+    feedsByCategoryList,
+    getOPMLBlob,
+    selectAllExportedFeeds,
+    exportMessages as t,
+    toggleExportedCategory,
+    toggleExportedFeed
+  } from '@slowreader/core'
+  import { onMount } from 'svelte'
+
+  import Button from '../../ui/button.svelte'
+  import Card from '../../ui/card.svelte'
+  import Loader from '../../ui/loader.svelte'
+  import RadioField from '../../ui/radio-field.svelte'
+  import FeedList from './list.svelte'
+
+  let currentFeeds: 'all' | 'select' = 'all'
+
+  function handleRadioChange(e: CustomEvent<'all' | 'select'>): void {
+    currentFeeds = e.detail
+    if (currentFeeds === 'all') {
+      selectAllExportedFeeds()
+    }
+  }
+
+  function handleSubmit(): void {
+    let blob = getOPMLBlob()
+
+    let url = URL.createObjectURL(blob)
+    let a = document.createElement('a')
+    a.href = url
+    a.download = 'feeds.opml'
+    a.click()
+    URL.revokeObjectURL(url)
+  }
+
+  onMount((): void => {
+    if (currentFeeds === 'all') {
+      selectAllExportedFeeds()
+    }
+  })
+</script>
+
+<h2>{$t.chooseTitle}</h2>
+<form on:submit|preventDefault={handleSubmit}>
+  <Card>
+    <RadioField
+      current={currentFeeds}
+      label={$t.type}
+      values={[
+        ['all', $t.allFeeds],
+        ['select', $t.selectFeeds]
+      ]}
+      on:change={handleRadioChange}
+    />
+  </Card>
+  <FeedList
+    disabled={currentFeeds === 'all'}
+    feedsByCategory={Array.from($feedsByCategoryList)}
+    selectedCategories={Array.from($exportedCategories)}
+    selectedFeeds={Array.from($exportedFeeds)}
+    on:toggleCategory={e => {
+      toggleExportedCategory(e.detail.categoryId)
+    }}
+    on:toggleFeed={e => {
+      toggleExportedFeed(e.detail.feedId, e.detail.categoryId)
+    }}
+  />
+  <div class="feeds-opml_submit">
+    <Button disabled={$creating} type="submit">{$t.submitOPML}</Button>
```

Comment:
Button doesn’t do nothing.

1. Latest Firefox
2. Open https://preview-192---staging-web-jfj4bxwwxq-ew.a.run.app/
3. Create account
4. Add `github.blog`
5. Open Export
6. Click OPML
7. Press Export to OPML
8. The button is clickable, but nothing happens